### PR TITLE
Fixes #15 - Adds ESLint

### LINKED ISSUE #15: Add linting to match Django's style

### PR DESCRIPTION
**Pull request by erosselli**
Fixes #15 - Adds ESLint along with relevant `.eslintrc` file (copied from the `django` repo so we have the same config).

All the changes in the JS files are just the fixes so linter is passing.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "rules": {
+        "camelcase": ["off", {"properties": "always"}],
```
Comment (erosselli) [inline .eslintrc:3]: Tbh the camelcase turned off makes me a bit sad 😢 but I guess we should stick with the original config?

Comment (knyghty) [reply .eslintrc:3]: I feel the same way on both counts.

**Review by knyghty — CHANGES_REQUESTED** [review]
Thanks! Looks almost perfect. Thanks for the help working on this.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      "lint": "eslint ."
+    },
+    "engines": {
+      "npm": ">=1.3.0"
+    },
+    "devDependencies": {
+      "eslint": "^8.56.0"
+    }
+  }
+
```
[the remark this answers is not in the document]
Comment (erosselli) [reply package.json]: Done :)

**Review by knyghty — APPROVED** [review]

**Review by knyghty — APPROVED** [review]
Thanks!

**Pull request merged**